tests/Unit/Http/Controllers/PageControllerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Locations\Tests\Unit\Http\Controllers;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tipoff\Authorization\Models\User;
+use Tipoff\Locations\Models\Location;
+use Tipoff\Locations\Models\Market;
+use Tipoff\Locations\Tests\TestCase;
+
+class PageControllerTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /** @test */
+    public function single_market_single_location()
+    {
+        $this->actingAs(User::factory()->create());
+
+        $location = Location::factory()->create();
+        $market = $location->market;
+
+        $this->get($this->webUrl("{$market->slug}"))
+            ->assertRedirect('/');
+
+        $this->get($this->webUrl("{$market->slug}/{$location->slug}"))
+            ->assertRedirect('/');
+    }
```

Comment:
Perfect. This is exactly how it should function with only one market & one location.